src/Language/PureScript/Publish/ErrorsWarnings.hs

```diff
@@ -182,10 +182,10 @@ displayUserError e = case e of
       , spacer
       ] ++ spdxExamples ++
       [ spacer
-      , para (concat
-          [ "Note that distributing code without a license means that nobody "
-          , "will (legally) be able to use it."
-          ])
+      , para (
```

Comment:
Does HLint not warn for the other instances of `concat` + a list literal? Changing this makes it inconsistent within the file. I don't really mind which approach we take but I do want it to be consistent within this file.